crowdsourcing/models.py

```diff
@@ -298,6 +298,12 @@ class Task(models.Model):
 class TaskWorker(models.Model):
     task = models.ForeignKey(Task, related_name='task_workers')
     worker = models.ForeignKey(Worker)
+    statuses = ((1, 'In Progress'),
```

Comment:
fair point, this would move to another model then, here we just want to keep track of who is working on the tasks